redbpf/src/load/map_io.rs

```diff
@@ -69,3 +69,38 @@ impl Stream for PerfMessageStream {
         Some(self.read_messages()).into()
     }
 }
+
+pub struct RingBufMessageStream {
```

Comment:
Is it necessary to export this to public?
Is it technical requirement to implement this feature or do you mean this to be used by users directly?

Reply:
A ring buffer is handy to be consumed through a Stream, so I think it's worth making this public. See the PerfMessageStream above, which was designed specifically around this usecase.